cmd/apiregister-gen/generators/controller_generator.go

```diff
@@ -114,6 +114,11 @@ func (c *{{.Target.Kind}}Controller) GetName() string {
 	return c.Name
 }
 
+// change the retry time at runtime
+func (c *{{.Target.Kind}}Controller) SetRetryTimes(r int) {
```

Comment:
rather than a setter, plz add another constructor func to avoid thread-safety issue, like `NewBucketControllerWithMaxRetries`

Reply:
Got it. It would be better to do this. It will be user's responsibility to add such a function. And it will be unnecessary to add function here.
My fault and will close it.
Thanks!